app/test_app/functional/me_api/test_me_credit_transfer.py

```diff
@@ -16,6 +20,42 @@ def test_get_me_credit_transfer_api(test_client, create_credit_transfer, create_
     assert response.status_code == 201
     assert response.json['data']['credit_transfers'][0]['id'] is create_credit_transfer.id
 
+def test_valid_create_me_credit_transfer_api(
```

Comment:
This test isn't really relevant to this PR, but it's something that really should be tested